### DISCUSSION TITLE
Problem in `Traits\SpatieTranslatable\Sluggable`

### DISCUSSION THREAD

**Discussion by EG-Mohamed** [Laravel-Backpack/community-forum#350] [Q&A (Help)] [answered]
Hello, I have a problem with `Sluggable` `Traits\SpatieTranslatable\Sluggable` it's throwing an error 
![image](https://user-images.githubusercontent.com/23424932/210341342-d3c3c344-28d3-4a00-b346-0e8918a4b0bd.png)
I've searched a lot and I found something about InnoDB Version mine is 5.7.24 but unfortunately I can't update it am using mapp pro for windows, what do I have to do to fix this problem?

here's a SQL error after dump the query 
![image](https://user-images.githubusercontent.com/23424932/210341833-b1c65e2f-0fd0-4b11-8157-1f7e6e6aa1b1.png)

**Comment by pxpm** [comment, marked as answer]
Hello @EGYWEB-Mohamed sorry for the late reply. 🙏 

I think you can't easily fix it. Sorry 😞 
Either your db version does not support `json` columns (or operations with json columns even if json is supported), or it could be due to `collation` in database and/or engine (innodb/myisam etc). 

Try asking uncle google, I am sorry I can't be more helpful. 

Anyway, from personal experience, the boost from 5.7 to 8.0 in mysql performance was something from other world, it's 100% worth the update. 

Cheers
